x-pack/plugins/security_solution/public/common/components/wrapper_page/index.tsx

```diff
@@ -38,18 +43,20 @@ Wrapper.displayName = 'Wrapper';
 
 interface WrapperPageProps {
   children: React.ReactNode;
-  className?: string;
   restrictWidth?: boolean | number | string;
   style?: Record<string, string>;
   noPadding?: boolean;
+  noTimeline?: boolean;
```

Comment:
instead of passing another prop, you can use https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/utils/timeline/use_show_timeline.tsx to see if you are showing or not the timeline. It will be consistent if you do it like that.

Reply:
Why introducing such implicit coupling with globals and all the apps in a purely presentational component? It seems against all the good practices of writing reusable and testable components.

Reply:
I will create a separate PR for this and express my concerns there and we can move on from there.